repos/fdbt-site/tests/pages/home.test.tsx

```diff
@@ -5,11 +5,11 @@ import Home from '../../src/pages/home';
 describe('pages', () => {
     describe('home page', () => {
         it('should render correctly', () => {
-            const tree = shallow(<Home multipleOperators />);
+            const tree = shallow(<Home multipleOperators isOnTestEnvironment />);
             expect(tree).toMatchSnapshot();
         });
         it('should render correctly with no multiple operators', () => {
-            const tree = shallow(<Home multipleOperators={false} />);
+            const tree = shallow(<Home multipleOperators={false} isOnTestEnvironment={true} />);
```

Comment:
`isOnTestEnvironment` as on line 8 is the same as `isOnTestEnvironment={true}` here. Are you trying to test both scenarios? In which case one of these needs to be `false`

Reply:
I think we said because this was a temporary thing (until the first bit of global settings makes its way into prod), we wouldn't bother with a unit test. I'm happy to add a test if you like :)